content/blog/day-2-kubernetes-migrating-eks-nodegroups-with-zero-downtime/index.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "Day 2 Kubernetes: Migrating EKS Node Groups with Zero Downtime"
+authors: ["mike-metral"]
+tags: ["Kubernetes","EKS", "Migrations", "Node Groups"]
+date: "2019-07-22"
```

Comment:
```suggestion
date: "2019-07-23"
```

Reply:
Oops, forgot to save. Done and changes pushed.